scrapy_fake_useragent/providers.py

```diff
@@ -36,31 +39,33 @@ def get_random_ua(self):
 
 
 class FakeUserAgentProvider(BaseProvider):
-    '''
-    Provides a random, real-life set of UA strings, powered by the fake_useragent library
-    '''
+    """
+    Provides a random, real-world set of UA strings, powered by the fake_useragent library.
+    """
+
     DEFAULT_UA_TYPE = 'random'
+
     def __init__(self, settings):
         BaseProvider.__init__(self, settings)
 
         self._ua_type = settings.get('FAKE_USERAGENT_RANDOM_UA_TYPE', self.DEFAULT_UA_TYPE)
 
         fallback = settings.get('FAKEUSERAGENT_FALLBACK', None)
```

Comment:
So, now that we do not catch the `AttributeError`, we maybe can default this to `self.DEFAULT_UA_TYPE` also?

Reply:
I think we have to have to have the fallback set separately as, if fake-useragent fails to reach the useragentstring.com or a cached server, it would fail for the "random" UA as well. 

But, we can probably remove the FAKEUSERAGENT_FALLBACK completely and let it use default Scrapy's `USER_AGENT`. But, I am not 100% sure, because we have a separate provider for this purpose.

Thoughts?

Reply:
When useragentstring.com is down fake-useragent won't be able to create the UserAgent() object. In that case, the FakeUserAgentProvider class won't even be used because this exception will be detected. We can have a fallback that it behaves as the fake-useragent fallback but I would default it to something that surely works, like "random"

Reply:
> When useragentstring.com is down fake-useragent won't be able to create the UserAgent() object

If I understand it correctly, if fallback is provided, `UserAgent` object will be instantiated ([reference](https://github.com/hellysmile/fake-useragent/blob/master/fake_useragent/fake.py#L91)) as the exceptions raised during requesting useragentstring.com would be surpressed. 

`random` requires fake-useragent useragentstring.com to be up as well as it uses `data_randomize` data structure which is filled up with browsers during useragentstring.com loading and parsing ([reference](https://github.com/hellysmile/fake-useragent/blob/master/fake_useragent/fake.py#L131)).

In any case, I think what we have now should work and be a good workaround for the package users.